### PR TITLE
1895 addressing UX review

### PR DESCRIPTION
**Pull request by sutt0n**
- select scanner button color
- tab colors
- scan preview background
- create case verbiage
- heading, description, and form gap

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -153,6 +153,7 @@
     "seed:s3": "rm -rf storage/s3/noop-documents-local-us-east-1 && cp -R storage/fixtures/s3/ storage/s3/noop-documents-local-us-east-1",
     "start:api": "./web-api/run-local.sh",
     "start:client:cypress": "CYPRESS=true npm run start:client",
+    "start:client:no-scanner": "NO_SCANNER=true npm run start:client",
```
Comment (matthopson) [inline package.json:156]: Just curious, when would we use this?

**Review by matthopson — APPROVED** [review]
🇫🇮

File: package.json (hunk 1 above)
Comment (sutt0n) [reply package.json:156]: When we want to run the UI without dynamsoft running, and we want to scan -- e.g. when there's a future story that has a flow to create a case internally.

Comment (matthopson) [reply package.json:156]: Ah, so we _must_ use this locally to test scanning. Right? Might be worth putting in the Readme in one of your PRs if it's not already there (part of DOD, I guess, anyway).

File: web-client/src/views/ScanBatchPreviewer.jsx [hunk 2]
```diff
@@ -430,32 +430,32 @@ export const ScanBatchPreviewer = connect(
...
+                >
+                  Change
+                </button>
+              </div>
             </div>
           </div>
         </div>
-        <div style={{ border: '1px solid #AAA', padding: '20px' }}>
+        <div
+          style={{ border: '1px solid #AAA', padding: '20px' }}
```
Comment (kkoskelin) [inline web-client/src/views/ScanBatchPreviewer.jsx]: we can now remove these inline styles, right? covered by class def?

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Just a bit of inline styles cleanup, but 👍 once addressed.

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**